docs/tools/buildkit.md

```diff
@@ -135,6 +108,84 @@ If you want to ensure that the buildkit CLI tools are always available, then:
     Each time you open a new terminal while working on Civi development, you would need to re-run the `export` command.
 
 
+### Configuring `amp` {:#amp-config}
+
+Buildkit provides a tool called `amp` which [civibuild](/tools/civibuild.md) uses when it needs to set up a new site. Before you can use `civibuild`, need to configure `amp` by telling it a bit about your system (e.g. what webserver you're using). 
+
+#### Interactive config
+
+```
+$ amp config
+```
+
+!!! tip "tips"
+    * Run this as a non-`root` user who has `sudo` permission. This will ensure that new files are owned by a regular user, and (if necessary) it enables `civibuild` to restart your webserver and edit `/etc/hosts`.
+    * Pay close attention to any instructions given in the output of this command.  They may involve adding a line to your Apache or nginx configuration file.
+    * To check which version of apache you have, run `apachectl -v`
+
+#### Testing amp's configuration {:#amp-test} 
+
+Test that `amp` is correctly configured.
+
+```
+$ amp test
+```
+
+!!! failure "Troubleshooting errors from `amp test`"
+    
+    * Try manually adding settings to your sebserver, as described below.
+    * Re-run `amp config`.
+
+#### Manually adding settings to your webserver {:#amp-webserver}
+
+During one of the steps in the `amp config` process, `amp` will *attempt to* alter the system-wide settings for your local webserver (apache or nginx). On some platforms, `amp` is not able to perform the necessary configuration and you must do it manually by following these steps:
```

Comment:
Yup. Currently the `amp config` wizard just displays a handful of possibilities and moves on to the next step -- https://github.com/amp-cli/amp/blob/master/src/Amp/Command/ConfigCommand.php#L139

In an ideal world, that script would stop and ask the user what to do. Depending on what config files/directories are around, it would present a menu like:

```
[1] Include amp in /etc/apache2/httpd.conf
[2] Include amp in /etc/apache2/conf.d/amp.conf
[3] Include amp in /opt/xampp/etc/httpd.conf
[4] Skip this step. I'll configure it manually.
```

If they choose 1-3, it would call `sudo` and try to update the selected file.

This way, it's harder for a user to ignore the step, but we still rely on their judgment for where/how to make the change.